code/modules/projectiles/guns/magic/staff.dm

```diff
@@ -105,4 +105,69 @@
 	max_charges = 6
 	recharge_rate = 4
 
+/obj/item/gun/magic/staff/flying
+	name = "staff of flying"
+	desc = "An artefact that spits bolts of graceful magic that can make something fly."
+	fire_sound = 'sound/magic/staff_change.ogg'
+	ammo_type = /obj/item/ammo_casing/magic/flying
+	icon_state = "staffofchange"
+	item_state = "staffofchange"
+
+/obj/item/gun/magic/staff/grounding
+	name = "staff of grounding"
+	desc = "An artefact that spits bolts of dense magic that hates flying things."
```

Comment:
see above

Reply:
Remove the "s" in "hates", since that presumably refers to the bolts, not the staff.

Reply:
need to do the second thing